### ISSUE TITLE
WCAG 2.2 Violation - There are instances of controls undergoing a programmatic change of state where the difference in state is not conveyed visually.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#18]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

#### Further information:

1. 

### Instances

- #33
